xkcd_dl/cli.py

```diff
@@ -216,6 +216,7 @@ def download_one(xkcd_dict, xkcd_num):
                         os.rename(file_name, "{description}.jpeg".format(
                             description=new_description)
                         )
+            os.chdir(WORKING_DIRECTORY)
```

Comment:
Yes. During testing, if the download succeeds, it does not return to the `WORKING_DIRECTORY`.    And CWD is `WD/xkcd_dl/1000/`. However, it if the comic is already downloaded, the CWD does not change.